lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -543,10 +549,12 @@ export class UserAgentApplication {
             if (interactionType === Constants.interactionTypeRedirect) {
                 if (!isLoginCall) {
                     this.cacheStorage.setItem(`${TemporaryCacheKeys.STATE_ACQ_TOKEN}${Constants.resourceDelimiter}${request.state}`, serverAuthenticationRequest.state, this.inCookie);
+                    this.logger.verbose("State cached for redirect");
```

Comment:
Did we choose to not log state as PII?

Reply:
For some reason, I thought that if we weren't actually printing out the state that Pii wasn't necessary, but I can change it so that any reference to state is Pii.

Reply:
> For some reason, I thought that if we weren't actually printing out the state that Pii wasn't necessary

This is correct. I think @pkanher617 is specifically asking about logging the state value itself (in addition to the log message you added).

Reply:
Ah, that makes sense. I chose to just log that message above, as it didn't seem like logging the state as Pii would be helpful in the logs. What do you think, @pkanher617 @jasonnutter?

Reply:
@pkanher617 @jasonnutter Added Pii logs for state being cached